react/components/Backdrop.tsx

```diff
@@ -27,12 +29,18 @@ const Backdrop: React.FC<Props> = props => {
     }
   }
 
-  const rootClasses = classnames(styles.backdrop, 'o-50 bg-base--inverted')
-
   return (
-    <div className={rootClasses} onClick={handleClick}>
-      {children}
-    </div>
+    <Fade in={open} timeout={transitionDuration}>
+      <div className={styles.backdropContainer}>
```

Comment:
I created this extra div because the `Fade` component uses opacity to create the animation in the root element of its child, so the way that it was before at the end of the transition the `Backdrop` would have `opacity: 1` always